dds/DCPS/transport/framework/TransportClient.h

```diff
@@ -180,8 +180,15 @@ class OpenDDS_Dcps_Export TransportClient
     TransportImpl::ConnectionAttribs attribs_;
 
     PendingAssoc()
-      : active_(false), removed_(false), blob_index_(0)
-    {}
+      : active_(false)
+      , removed_(false)
+      , blob_index_(0)
+    {
```

Comment:
I contemplated this.  Actually I initially coded it up that way, but noticed that many of the structs we create were not being provided constructors keeping them as simple as possible.  I'll go back to the constructor, I agree that it would be cleaner.